openssl-dynamic/src/main/c/ssl_private.h

```diff
@@ -213,6 +217,11 @@ struct tcn_ssl_ctxt_t {
     apr_uint32_t            ticket_keys_renew;
     // The client presented a ticket that did not match any key in the list.
     apr_uint32_t            ticket_keys_fail;
+
+    jmethodID               verifier_method;
+    jmethodID               cert_requested_callback_method;
```

Comment:
@Scottmitch did you change the ordering for any reasons like memory savings due layout ?